### PR TITLE
RangePicker support time mode

### PR DESCRIPTION
**Pull request by susiwen8**
This PR is related to https://github.com/ant-design/ant-design/issues/16492

### CODE REVIEW

**Comment by zombieJ** [comment]
加一个 test case？

**Comment by susiwen8** [comment]
@zombieJ 加上了。

File: src/RangeCalendar.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class RangeCalendar extends React.Component {
       firstSelectedValue: null,
       hoverValue: props.hoverValue || [],
       value,
-      showTimePicker: false,
+      showTimePicker: props.mode && props.mode.toString() === 'time,time' || false,
```
Comment (zombieJ) [inline src/RangeCalendar.js]: 下面已经会更新 mode 了，这边应该不需要管 `props.mode` 了才对

Comment (susiwen8) [reply src/RangeCalendar.js]: 没错，当时图省事了......

**Pull request merged**